h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -160,6 +173,124 @@ public long getModelCount() {
     return _hyperSpaceWalker.getMaxHyperSpaceSize();
   }
 
+  private class ModelFeeder<MP extends Model.Parameters> {
```

Comment:
I think this class is long enought and this does not need to be inner class

Reply:
Agreed. Will do.

Reply:
It uses a lot from the grid itself. I don't want to externalize it because of that. I think it's a justified case for inner class just because of the amount of  reuse and maybe more importantly it's limited context to one place.